Update/kage_010.txt

```diff
@@ -731,25 +731,25 @@ void main()
 //rそうだ。誰も頼りになんかならない。助けてなんかくれない。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "そうだ。誰も頼りになんかならない。助けてなんかくれない。",
-			NULL, "Right. Nobody can rely on that. Don't ask for help.", Line_Normal);
+			NULL, "Right. I can't rely on anyone. No one will help me.", Line_Normal);
 	ClearMessage();
 
 //r『アイツ』がわめきたてる恐怖におののき、疑って嫌悪しながらも不安のあまり完全には拒絶できず、野放しにし続けてきたんだ。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "『アイツ』がわめきたてる恐怖におののき、疑って嫌悪しながらも不安のあまり完全には拒絶できず、野放しにし続けてきたんだ。",
-			NULL, "No matter how much <i>that person</i> trembles in fear, I don't think I can refuse this, so I'll continue to leave it unleashed.", Line_Normal);
+			NULL, "I've been leaving <i>that thing</i> to her own devices because no matter how much I feared her rants and doubted and was disgusted by her, I was too uneasy to completely reject her.", Line_Normal);
```

Comment:
```suggestion
			NULL, "I've been leaving <i>that thing</i> to her own devices because no matter how much I feared her rants, and felt doubt and disgust toward her, I was too uneasy to completely reject her.", Line_Normal);
```